.deepsource.toml

```diff
@@ -0,0 +1,16 @@
+version = 1
+
+test_patterns = ["t/**"]
+
+exclude_patterns = [
+    "requirements/",
+    "extra/**",
+    "examples/**"
+]
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```

Comment:
I think this file should go in separate PR. There should be decision that we want to use deepsource and it should be configured to trigger it by PR.

Reply:
No worries, I'll remove the file.
Do you want me to open an issue for discussion regarding activating DeepSource?

Reply:
@akshgpt7 Please do.
I'm using DeepCode in another project.
Are they a direct competitor of yours?

Reply:
@thedrow There are various added benefits that DeepSource offers over other code analysis tools like DeepCode and Codacy:
- Automatic code formatting to comply with the code style (through Transformers).
- Option to Autofix detected issues through one click.
- DeepSource Python analyzer detects 520 issues as of now (new support being continuously added).
- Less than 5% false positive reports in DeepSource.
- Option to report false positives directly to the team.

These are of course some of the benefits, just for you to assess if the tool is right for the project :)
If you have any questions, feel free to reach out or have a look at the [docs](https://deepsource.io/docs).

Reply:
Also, I've removed the config file, and opened an issue #1329.